### PR TITLE
Feature/backend/api/route actions unread read

### PR DESCRIPTION
**Pull request by sapiens-sapide**
API v2 handles actions "set_read" & "set_unread".

Need to update vendor.json of go packages after this PR will be merge into master.

### CODE REVIEW

File: src/backend/configs/caliopen-go-api_dev.yaml [hunk 1]
```diff
@@ -5,20 +5,25 @@ ApiConfig:
...
     password: ""      #no password set
     db: 0             #use default db
 ProxyConfig:
-  host: 127.0.0.1
+  host: 0.0.0.0
   port: 31415
   routes:
   - /api/v2/: 127.0.0.1:6544
-  - /: 127.0.0.1:6543         # route "/" catch all other requests
+  - /: apiv1.dev.caliopen.org:6543         # route "/" catch all other requests
```
Comment (gdchamal) [inline src/backend/configs/caliopen-go-api_dev.yaml]: no one have this host apiv1 defined as far I know.

File: src/backend/main/go.backends/RESTInterfaces.go [hunk 2]
```diff
@@ -12,4 +12,9 @@ type APIStorage interface {
 	UserNameStorage
 	GetMessage(user_id, msg_id string) (msg *obj.Message, err error)
 	GetLocalsIdentities(user_id string) (identities []obj.LocalIdentity, err error)
+	SetMessageUnreadStatus(user_id, message_id string, status bool) error
```
Comment (gdchamal) [inline src/backend/main/go.backends/RESTInterfaces.go]: we know that unread is a status, I think SetMessageUnread is correct for naming

File: src/backend/main/go.backends/index/elasticsearch/messages.go [hunk 3]
```diff
@@ -42,3 +42,14 @@ func (es *ElasticSearchBackend) IndexMessage(msg *objects.Message) error {
...
 
 }
+
+func (es *ElasticSearchBackend) SetMessageUnreadStatus(user_id, message_id string, status bool) (err error) {
+	payload := struct {
+		Is_unread bool `json:"is_unread"`
+	}{status}
+
+	update := es.Client.Update().Index(user_id).Type("indexed_message").Id(message_id)
+	_, err = update.Doc(payload).Refresh("true").Do(context.TODO())
```
Comment (gdchamal) [inline src/backend/main/go.backends/index/elasticsearch/messages.go]: a TODO context ? a bit strange ;)

Comment (sapiens-sapide) [inline src/backend/main/go.backends/index/elasticsearch/messages.go]: it's an empty context, required by ES library.

File: src/backend/main/go.backends/RESTInterfaces.go (hunk 2 above)
Comment (sapiens-sapide) [inline src/backend/main/go.backends/RESTInterfaces.go]: OK, I'll shorten the func name

File: src/backend/configs/caliopen-go-api_dev.yaml (hunk 1 above)
Comment (sapiens-sapide) [inline src/backend/configs/caliopen-go-api_dev.yaml]: copied from docker file config.

**Pull request merged**